cli/src/v2-blueprint.js

```diff
@@ -20,26 +20,29 @@ export async function installV2Blueprint(info) {
     : info.isNpm
     ? '--npm'
     : '';
-  let ts = info.isTS ? '--typescript' : '';
+  let ts = info.isTS ? '--typescript' : undefined;
 
   // we'll create the addon here instead of using a directory based on the actual name, as this is temporary anyway
   // and prevents issues with scoped package names getting in some way dasherized by ember-cli
   let addonDir = 'addon';
 
-  await execaCommand(
-    `npx ember-cli@^4.10.0 addon ${info.name}` +
-      ` --dir=${addonDir}` +
-      ` --blueprint @embroider/addon-blueprint` +
-      // ` --blueprint ../addon-blueprint` +
-      ` --test-app-location=${info.testAppLocation}` +
-      ` --test-app-name=${info.testAppName}` +
-      ` --addon-location=${info.addonLocation}` +
-      ` ${packager}` +
-      ` ${ts}` +
+  await runEmber(
+    [
+      'addon',
+      info.name,
+      `--dir=${addonDir}`,
+      // needs https://github.com/embroider-build/addon-blueprint/pull/137 for proper TS support
+      '--blueprint=embroider-build/addon-blueprint#update-for-upstream-ts-changes',
```

Comment:
- [ ] Use a new published version once we have https://github.com/embroider-build/addon-blueprint/pull/137 merged.

Reply:
Reverted that, it's now using the npm published blueprint again, which as of https://github.com/embroider-build/addon-blueprint/releases/tag/v1.7.0 contains all the things we need

Reply:
Yay!